HIP/hip-external-transaction-signing.md

```diff
@@ -0,0 +1,337 @@
+- hip: &lt;HIP number (this is determined by the HIP editor)>
+- title: External Transaction Signing for SDK and other clients
+- author: Danno Ferrin (danno.ferrin@hedera.com)
+- type: Standards Track
+- category: Application
+- status: Draft
+- created: 19 Oct 2021
+- discussions-to: &lt;a URL pointing to the official discussion thread>
+- updated: &lt;comma separated list of dates>
+- requires: hip-caip-identifiers
+- replaces: &lt;HIP number(s)>
+- superseded-by: &lt;HIP number(s)>
+
+## Abstract
+
+An RPC protocol is described that would allow clients and SDKs to sign
+transactions with external programs, devices, and services.
+
+## Motivation
+
+The current Hedera SDK requires that the private keys for accounts be available
+and signed in process. One process is to provide the private key in an
```

Comment:
Can this be a bit more specific? the HAPI does not require private keys, and some of the sdks already provide mechanisms to sign transactions without receiving a private key.

Reply:
Only the env methods are documented.  While they provide hooks there are no default implementations.  The goal of this HIP is to start standardizing how those implementations might work.